src/components/application_manager/test/commands/mobile/reset_global_properties_test.cc

```diff
@@ -308,7 +337,7 @@ TEST_F(ResetGlobalPropertiesRequestTest,
   (*msg_)[am::strings::msg_params][am::strings::properties][0] =
       mobile_apis::GlobalProperty::TIMEOUTPROMPT;
   (*msg_)[am::strings::msg_params][am::strings::properties][1] =
-      mobile_apis::GlobalProperty::MENUICON;
+      mobile_apis::GlobalProperty::MENUNAME;
```

Comment:
@LuxoftAKutsan Well this change here is not critical: presence of either MENUICON or MENUNAME was nesessary to send ui notification which was needed for specific case in on_event function, MENUICON realization used to be empty, so it was used here to create empty ui notification. I changed it to MENUNAME since it has slightly shorter realization (over MENUICON, which is now not empty).